packages/commons-ui-core/src/NavList/index.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import * as React from "react";
+
+import NavIcon from "./NavIcon";
+import NavMenu from "./NavMenu";
+
+function NavList({ menu, links }) {
+  if (!menu?.length) {
+    return null;
+  }
+  if (!links?.length) {
+    return null;
+  }
+  return (
+    <NavMenu menu={menu}>
+      <NavIcon links={links} />
```

Comment:
This is optional e.g. `Footer` uses the same `NavList` but doesn't have Twitter button.